quickjs-libc.c

```diff
@@ -94,6 +94,10 @@ extern char **environ;
 #include "list.h"
 #include "quickjs-libc.h"
 
+#ifndef QJS_NATIVE_MODULE_SUFFIX
+#define QJS_NATIVE_MODULE_SUFFIX ".so"
```

Comment:
Why did you go back on having different suffixes on windows? I think it makes sense to use the platform default.

Reply:
@saghul 
[test_point.js](https://github.com/quickjs-ng/quickjs/blob/master/examples/test_point.js) use esm `import`
but [esm](https://nodejs.org/api/esm.html#import-specifiers) not support dynamic splicing.
```js
import * as os from "os";
const isWin = os.platform === 'win32';
import { fib } from `./fib.${isWin ? 'dll' : 'so'}`;

console.log("Hello World");
console.log("fib(10)=", fib(10));

```
``` sh
SyntaxError: string expected
    at ./examples/test_fib.js:4:1
```

Reply:
You can use a dynamic import and await it.

Reply:
@saghul ok, this is [change](https://github.com/quickjs-ng/quickjs/pull/490/commits/fedc1bc8ede6d18f2f78c394c66eee6564745f3a)